Formula/wandio.rb

```diff
@@ -3,7 +3,8 @@ class Wandio < Formula
   homepage "https://research.wand.net.nz/software/libwandio.php"
   url "https://research.wand.net.nz/software/wandio/wandio-4.2.3.tar.gz"
   sha256 "78c781ce2c3783b85d894e29005b7e98fc246b33f94616047de3bb4d11d4d823"
-  license "GPL-3.0"
+  license "LGPL-3.0-or-later"
```

Comment:
License according to homepage:

> **NEW: From libwandio 1.0.4 onwards, the libwandio source code is licensed under the LGPL v3.**
> 
> All releases of libwandio prior to 1.0.4 are licensed under the GPL v2.

Though some of files still show GPL-2.0 or later

Reply:
Do they indicate whether it's `-or-later` or `-only`?

Reply:
Only saw it in parts of available code, e.g.
https://github.com/wanduow/wandio/blob/4.2.3-1/lib/curl-helper.c#L11-L14
```
 * libwandio is free software; you can redistribute it and/or modify
 * it under the terms of the GNU Lesser General Public License as published by
 * the Free Software Foundation; either version 3 of the License, or
 * (at your option) any later version.
```